src/jestSnapshotResolver/snapshotResolver.js

```diff
@@ -0,0 +1,32 @@
+import manifest from "../custom/manifest.json";
+
+const versionFolder = `__${manifest.name}-snapshots__/`;
+/**
```

Comment:
Not a real problem, just a question: typescript isn't an option here?

Reply:
changing to ts file worked